res/css/structures/_GroupView.scss

```diff
@@ -368,6 +368,61 @@ limitations under the License.
     padding: 40px 20px;
 }
 
+.mx_GroupView_spaceUpgradePrompt {
+    padding: 16px 50px;
+    background-color: $header-panel-bg-color;
+    border-radius: 8px;
```

Comment:
I suspect this prompt probably doesn't need a unique border radius - should we have SCSS variables for standard styling of this?

Reply:
Yup, probably worth to put down into the bucket of tasks for implementing Compound in react-sdk